instructure/canvas-android
# [MBL-16931][Student] K5 Grades LGO

Test plan: 
- Verify that no quantitative data is shown on K5 grades when quantitative data is restricted.
- The progress should be hidden as well.
- Verify that everything works the same when quantitative data is not restricted.
- There was a slight API change what we also implemented in the Parent app, test that as well.

refs: MBL-16931
affects: Student
release note: none

## Checklist
